kyo-core/shared/src/main/scala/kyo/Clock.scala

```diff
@@ -13,7 +13,7 @@ object Clock:
 
     private val local = Local.init(live)
 
-    def let[A, S](c: Clock)(f: => A < (IO & S))(using Frame): A < (IO & S) =
```

Comment:
Minor unrelated cleanup. This used to require `IO` because `Local` was implemented based on it but it's not the case anymore.